<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPerson</class>
 <widget class="QDialog" name="FormPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Участник</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>433</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ed_surname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ed_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Группа</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Команда</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Квалификация</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Время старта</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Номер чипа</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Комментарий</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTimeEdit" name="ted_start">
      <property name="displayFormat">
       <string>hh:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPlainTextEdit" name="pled_comment"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cb_qual">
      <property name="currentText">
       <string>б/р</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>б/р</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Iю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IIю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IIIю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>II</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>III</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>КМС</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>МС</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>МСМК</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ЗМС</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="ed_date"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cb_org"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_group"/>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="lb_check_cardNum">
      <property name="text">
       <string>*Проверка чипа*</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="ed_cardNum">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="ed_bib">
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormPerson</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormPerson</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
